src/network/netdev/bridge.c

```diff
@@ -287,6 +293,36 @@ int link_set_bridge(Link *link) {
         return r;
 }
 
+int config_parse_bridge_igmp_version(const char *unit,
+                                     const char *filename,
+                                     unsigned line,
+                                     const char *section,
+                                     unsigned section_line,
+                                     const char *lvalue,
+                                     int ltype,
+                                     const char *rvalue,
+                                     void *data,
+                                     void *userdata) {
+        Bridge *b = userdata;
+        uint8_t u;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou8(rvalue, &u);
+        if (r < 0 || !IN_SET(u, 2, 3)) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "bridge IGMP version number '%s' is invalid, ignoring assignment: %s", lvalue, rvalue);
```

Comment:
Inserting `lvalue` sounds strange for me.
How about the following:
`"Invalid bridge IGMP version number, ignoring assignment: %s", rvalue`
or
`"Invalid bridge IGMP version number '%s', ignoring assignment.", rvalue`

Reply:
updated thanks for the comment.